scripts/configure_shotgun

```diff
@@ -21,7 +22,9 @@ def config(env_script, server_cert):
     """Generates the Qiita configuration files"""
     if server_cert == 'None':
         server_cert = None
-    plugin.generate_config(env_script, 'start_shotgun',
+    kd_plugin.generate_config(env_script, 'start_shotgun',
```

Comment:
Although this may work for the configure case, it will not work for the `start_shotgun` case, different scripts will be needed. If that is the route, I would suggest creating 2 different repos, but I think it will be easier to have a single plugin here in which we add the different commands, rather than creating a plugin per command.